CHANGELOG.md

```diff
@@ -15,6 +15,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [The `Environment` is now automatically propagated through to nested `BlueprintViews` within a displayed `Element` hierarchy](https://github.com/square/Blueprint/pull/234). This means that if your view-backed `Elements` themselves contain a `BlueprintView` (eg to manage their own state), that nested view will now automatically receive the correct `Environment` across `BlueprintView` boundaries. If you were previously manually propagating `Environment` values you may remove this code. If you would like to opt-out of this behavior; you can set `view.automaticallyInheritsEnvironmentFromContainingBlueprintViews = false` on your `BlueprintView`.
 
+- [Lifecycle hooks][#244]. You can hook into 4 lifecycle events:
+  - `onMount` occurs when a backing view has been created and attached to its hosting view's view hierarchy
+  - `onAppear` occurs when a mounted element becomes visible
+  - `onDisappear` occurs when a visible element is no longer visible
+  - `onUnmount` occurs when a mounted element is removed from the hosting view's view hierarchy
+
+  Each hook is can be added with a modifier of the same name:
```

Comment:
```suggestion
  Each hook can be added with a modifier of the same name:
```

this never got fixed

Reply:
Oh geez, I think I accepted the suggested change on the web and then blew it away with a force push. 🤦